python/dgllife/utils/complex_to_graph.py

```diff
@@ -140,27 +140,36 @@ def PN_graph_construction_and_featurization(ligand_mol,
         ligand_atom_indices_left = list(range(ligand_mol.GetNumAtoms()))
         protein_atom_indices_left = list(range(protein_mol.GetNumAtoms()))
 
-    # Construct bigraph for stage 1
-    node_featurizer = CanonicalAtomFeaturizer(atom_data_field='h')
-    edge_featurizer = CanonicalBondFeaturizer(bond_data_field='e')
+    # Node featurizer for stage 1
+    atoms = ['H','N','O','C','P','S','F','Br','Cl','I','Fe','Zn','Mg','Cs','Na','Mn','Ca','Co','Ni','Se','Cu','Cd','Hg','Sr','K']
```

Comment:
I think this should be 
```
['C', 'N', 'O', 'S', 'F', 'P', 'Cl', 'Br', 'Mg', 'Na', 'Ca', 'Fe', 'I', 'K', 'Co', 'Se', 'Zn', 'H', 'Cu', 'Ni', 'Cd', 'Mn', 'Hg']
```

The first type of featurization in `CanonicalAtomFeaturizer` is `atom_type_one_hot`. By default, the list of atom types considered is [here](https://github.com/awslabs/dgl-lifesci/blob/master/python/dgllife/utils/featurizers.py#L145). Since "5, 13, 14, 16, 17, 19, 20, 21, 22, 23, 24, 27, 30, 31, 33, 36, 38, 39, 40, 42" did not appear, we just need to remove the corresponding atom types.

Reply:
The list of feaatures are derived from https://github.com/awslabs/dgl-lifesci/blob/master/examples/property_prediction/csv_data_configuration/analysis.py